cardano-crypto-class/src/Cardano/Crypto/Hash/Blake2b.hs

```diff
@@ -30,7 +30,7 @@ data Blake2b_256
 instance HashAlgorithm Blake2b_224 where
   type SizeHash Blake2b_224 = 28
   hashAlgorithmName _ = "blake2b_224"
-  digest _ = blake2b_libsodium (fromIntegral (natVal (Proxy::Proxy (SizeHash Blake2b_256))))
+  digest _ = blake2b_libsodium (fromIntegral (natVal (Proxy::Proxy (SizeHash Blake2b_224))))
```

Comment:
Oh good, 224 :-)

But I think we can still simplify down to just `blake2b_libsodium 224`

I'd suggest rebasing at the end to squash this fix into the earlier commit.